extensions/spring-data-rest/runtime/src/main/resources/META-INF/quarkus-extension.yaml.template

```diff
@@ -10,4 +10,4 @@ metadata:
   guide: "https://quarkus.io/guides/spring-data-rest"
   categories:
     - "compatibility"
-  status: "experimental"
+  status: "preview"
```

Comment:
@gytis do you agree with this?

Reply:
It shows tech preview on code.quarkus.redhat.com.

Reply:
If we do this, then we should also move the `rest-data-panache` because most of the logic comes from there.

Reply:
Makes sense.

Up to you if you think it's mature enough to be moved to `preview
`

Reply:
Is there criteria defined for this?

Reply:
The main criterion is whether one expects an extension to go away completely, or undergo massive changes.

If all you expect are potential underlying changes or small API changes, then you would likely move it to `preview`

Reply:
They're both marked as preview on code.quarkus.redhat.com

![Screen Shot 2021-05-17 at 9 03 39 AM](https://user-images.githubusercontent.com/363447/118493226-e1b2df80-b6ee-11eb-899a-e384c9c68de4.png)

Reply:
I'm not sure how does the mapping work there. It's still `experimental` in the template file https://github.com/quarkusio/quarkus/blob/main/extensions/panache/hibernate-orm-rest-data-panache/runtime/src/main/resources/META-INF/quarkus-extension.yaml.template#L16.

Reply:
Anyway, I think we can move them to preview.

Reply:
👍🏼

Reply:
My 2 cents. I think we should keep the `hibernate-orm-rest-data-panache` change separate from this ticket, since this is for the Spring extensions.

Reply:
Sure, we can do that separately.